docs/website/blog/2023-11-29-combining-sap-hana-and-snowflake-using-dlt.md

```diff
@@ -0,0 +1,76 @@
+---
+slug: dlt-dagster
+title: "Combining SAP HANA and Snowflake using dlt"
+image: https://dlt-static.s3.eu-central-1.amazonaws.com/images/sap_snowflake_blog_data_link_image.png
+authors:
+    name: Rahul Joshi
+    title: Data Science Intern at dltHub
+    url: https://github.com/rahuljo
+    image_url: https://avatars.githubusercontent.com/u/28861929?v=4
+tags: [SAP, SAP HANA, Snowflake, Cloud, ETL]
+---
+:::info
+**TL;DR: While most companies continue to build their businesses on top of SAP, when it comes to analytics, they prefer to take advantage of the price and elastic compute of modern cloud infrastructure. As a consequence, we get several dlt users asking for a simple and low-cost way to migrate from SAP to cloud data warehouses like Snowflake. In this blog, I show how you can build a custom SAP connector with dlt and use it to load SAP HANA tables into Snowflake.**
+:::  
+
+![Blog image](https://dlt-static.s3.eu-central-1.amazonaws.com/images/sap_snowflake_blog_data_link_image.png)
+
+In case you haven’t figured it out already, we at dltHub love creating blogs and demos. It’s fun, creative, and gives us a chance to play around with many new tools. We are able to do this mostly because, like any other modern tooling, dlt just *fits* in the modern ecosystem. Not only does dlt have existing [integrations](https://dlthub.com/docs/dlt-ecosystem) (to, for example, GCP, AWS, dbt, airflow etc.) that can simply be “plugged in”, but it is also very simple to customize it to integrate with almost any other modern tool (such as [Metabase](https://dlthub.com/docs/blog/postgresql-bigquery-metabase-demo), [Holistics](https://dlthub.com/docs/blog/MongoDB-dlt-Holistics), [Dagster](https://dlthub.com/docs/blog/dlt-dagster), [Prefect](https://dlthub.com/docs/blog/dlt-prefect) etc.). 
+
+But what about enterprise systems like SAP? They are, after all, the most ubiquitous tooling out there: according to SAP [data](https://assets.cdn.sap.com/sapcom/docs/2017/04/4666ecdd-b67c-0010-82c7-eda71af511fa.pdf), 99 out of 100 largest companies are SAP customers. A huge part of the reason for this is that their ERP system is still the gold standard in terms of effectivity and reliability. However, when it comes to OLAP workloads like analytics, machine learning, predictive modelling etc., [many companies prefer the convenience and cost savings of modern cloud solutions](https://www.statista.com/statistics/967365/worldwide-cloud-infrastructure-services-market-share-vendor/) like GCP, AWS, Azure, etc..
+
+So, wouldn’t it be nice to be able to integrate SAP into the modern ecosystem?
+
+Unfortunately, this is not that simple. Unlike modern tools, SAP does not integrate easily with non-SAP systems, and migrating data out from SAP is complicated and/or costly. This often means that ERP data stays separate from analytics data, resulting in data silos and preventing companies from making the most out of their analytics.
```

Comment:
delete "Unlike modern tools"

delete "resulting in data silos and preventing companies from making the most out of their analytics."